resources/wikia/modules/domCalculator.js

```diff
@@ -0,0 +1,25 @@
+/*global define*/
+define('wikia.domCalculator', [
+], function () {
+	'use strict';
+
+	function getTopOffset(element) {
+		var topPos = 0,
```

Comment:
Just a suggestion you could assign here `topPos = element.offsetTop` and change `do...while` loop below to `while` loop.